homeassistant/components/fritzbox/sensor.py

```diff
@@ -30,6 +29,7 @@
 )
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.util import dt
```

Comment:
```suggestion
from homeassistant.util.dt import utc_from_timestamp
```

Only import the needed method